Can't degrade_adc  for PD3/A4 (ch32v003)

#### Description
AdcChannel trait isn't implemented for PD3/A4
```rust
    let adc_channel_7 = p.PD4.degrade_adc(); // success
    let adc_channel_4 = p.PD3.degrade_adc(); // not available
```
```
error[E0599]: no method named `degrade_adc` found for struct `PD3` in the current scope
```
#### Affected devices:
- `ch32v003f4p6` and `CH32V003f4u6`

--- a/data/family/CH32V0.yaml
+++ b/data/family/CH32V0.yaml
@@ -570,7 +570,7 @@
       signal: IN2
     - pin: PD2
       signal: IN3
-    - pin: PC3
+    - pin: PD3
       signal: IN4
     - pin: PD5
       signal: IN5

